src/interface/desktop/chat.html

```diff
@@ -626,10 +626,9 @@
 
             let chatBody = document.getElementById("chat-body");
             chatBody.innerHTML = "";
-            let conversationId = chatBody.dataset.conversationId;
             let chatHistoryUrl = `/api/chat/history?client=desktop`;
-            if (conversationId) {
-                chatHistoryUrl += `&conversation_id=${conversationId}`;
+            if (chatBody.dataset.conversationId) {
+                chatHistoryUrl += `&conversation_id=${chatBody.dataset.conversationId}`;
```

Comment:
🤔 why not use the computed variable?

Reply:
Not any important reason, just didn't seem necessary to introduce a new variable given the value is just being used in 2 places